### PR TITLE
fix(curriculum): update user function to user object

### LINKED ISSUE #49418: "user function" was meant to be "user object"? (tests in "Serialization of a User Object")
### Describe the Issue

There are these tests in this lesson:

> You should serialize user function correctly.

> You should deserialize user function correctly.

I guess it was meant to be "user object", not "user function".

### Affected Page

https://www.freecodecamp.org/learn/quality-assurance/advanced-node-and-express/serialization-of-a-user-object

### Your code

```

```

### Expected behavior

The tests should be:
> You should serialize user object correctly.

> You should deserialize user object correctly.

### Screenshots

image

### System

- Device: Laptop
- OS: Windows 10
- Browser: Chrome
- Version: 110.0.5481.104

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MohitBansal321**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49418

### CODE REVIEW

**Comment by gikf** [comment]
Hey @MohitBansal321

please update the PR title, to follow convention from the contributing guidelines.

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/serialization-of-a-user-object.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Submit your page when you think you've got it right. If you're running into erro
 
 # --hints--
 
-You should serialize user function correctly.
+You should serialize user object correctly.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/serialization-of-a-user-object.md:53]:
```suggestion
You should serialize the user object correctly.
```

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/serialization-of-a-user-object.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ async (getUserInput) => {
 }
 ```
 
-You should deserialize user function correctly.
+You should deserialize user object correctly.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/serialization-of-a-user-object.md:73]:
```suggestion
You should deserialize the user object correctly.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by MohitBansal321** [comment]
@gikf now it is okay

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**